media/src/androidMain/kotlin/dev/icerock/moko/media/BitmapUtils.kt

```diff
@@ -49,5 +51,44 @@ object BitmapUtils {
         return Bitmap.createScaledBitmap(bitmap, width, height, true)
     }
 
+    fun getBitmapForStream(
+        inputStream: InputStream,
+        maxImageWidth: Int,
+        maxImageHeight: Int
+    ): Bitmap {
+        val bitmapOptions = BitmapFactory.Options().apply {
+            inJustDecodeBounds = true
+        }
+        val bitmapBytes = inputStream.readBytes()
+        BitmapFactory.decodeByteArray(bitmapBytes, 0, bitmapBytes.size, bitmapOptions)
+
+        bitmapOptions.apply {
+            inSampleSize = calculateInSampleSize(bitmapOptions, maxImageWidth, maxImageHeight)
+            inJustDecodeBounds = false
+        }
+        return BitmapFactory.decodeByteArray(bitmapBytes, 0, bitmapBytes.size, bitmapOptions)
```

Comment:
again decode stream...
just read bytes is read all file into memory (it's not so big like decoded bitmap, but it can be up to 16mb on current devices)

Reply:
`decodeByteArray` to `decodeStream` was replaced.